src/Be.Vlaanderen.Basisregisters.Projector/ConnectedProjections/RetryPolicy.cs

```diff
@@ -10,5 +12,11 @@ public static class RetryPolicy {
         public static MessageHandlingRetryPolicy LinearBackoff<TException>(int numberOfRetries, TimeSpan initialWait)
             where TException : Exception
             => new LinearBackOff<TException>(numberOfRetries, initialWait);
+
+        public static MessageHandlingRetryPolicy Configure(
```

Comment:
Maybe we should explicitly call this ConfigureLinearBackoff? Because we are now expecting a retry to have a number of retries and a delay in config. It is very possible other strategies will have more/less parameters. Making this fully generic will drive us too far.

So maybe just make this a specific one for linear backoff?